scripts/run_eks_smoke_test.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I would make this an explicit parameter. Changing the behavior of the script depending of where it's invoked from is kind of a gotcha.

Invoke it like this:
```lang=shell
$ ./scripts/run_eks_smoke_test.sh ./scripts/eks-smoke/test
```
And in this script you'd have:
```lang=shell
if [ $# -neq 1 ]
then
    echo "Usage: $0 path/to/yaml/"
    exit
fi
readonly yaml_dir=$1
...
kubectl delete -f $yaml_dir/kip-nginx.yaml
```